gslib/commands/rsync.py

```diff
@@ -513,8 +513,14 @@
 
                  will skip all .txt and .jpg files in dir.
 
-                 NOTE: When using this on the Windows command line, use ^ as an
+                 NOTE: When using this in the Windows cmd.exe command line interpreter, use ^ as an
                  escape character instead of \ and escape the | character.
+                 
+                   gsutil rsync -x ".*\.txt$^|.*\.jpg$"
```

Comment:
Sorry that I missed reviewing this way back when :-/

My regex is pretty weak, so I could be way off, but should this be ".*^.txt$^|.*^.jpg$" ?

Reply:
We probably don't need to list the example again, so this is fine if we remove lines 518 and 519.